src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -1017,9 +1017,7 @@
         },
         "file_monitor_ignored_components": {
             "type": "array",
-            "items": {
```

Comment:
Is it necessary to remove this? (seems like it provides useful validation if someone tries to stuff a non-string in there)

Reply:
Good point, added that back, and also added it to the example (my own) a few lines up that I was following.